python_modules/libraries/dagster-snowflake/dagster_snowflake/resources.py

```diff
@@ -2,6 +2,9 @@
 import warnings
 from contextlib import closing, contextmanager
 
+from sqlalchemy import create_engine
+from snowflake.sqlalchemy import URL
```

Comment:
For users who aren't using the sqlalchemy connector, it would be preferable to let them avoid depending on these packages.  Are we able to include these imports under the if statement where we need them?

Reply:
I moved these inside the "if" statement — let me know if that's what you had in mind.

Reply:
Perfect